site/vite.config.ts

```diff
@@ -17,4 +17,7 @@ export default defineConfig({
   ssr: {
     external: ['monaco-editor'],
   },
+  server: {
+    open: true,
+  },
```

Comment:
```suggestion
```

Reply:
OK~

Reply:
![image](https://user-images.githubusercontent.com/11926244/236848068-e627acdd-5cfe-49b2-ae1c-ce2070ad9a03.png)
Hi Flouse. In general, the turbo build command can sometimes time out or fail due to a variety of reasons, including network or CPU usage, as you mentioned.

One potential solution to this issue is to increase the timeout value using the --timeout flag. For example, you could try running turbo build index.html --timeout=60000 to increase the timeout to 60 seconds.

Reply:
### The failed deployment
https://vercel.com/flouse/ckbacademy/E7zv79j6XjbsybyQo3BXqg6YhHhy#L27